field/src/field.rs

```diff
@@ -54,6 +54,11 @@ pub trait AbstractField:
         self.clone() * self.clone()
     }
 
+    #[must_use]
+    fn cube(&self) -> Self {
+        self.clone() * self.clone() * self.clone()
```

Comment:
Nit, but maybe the default impl could be `self.clone() * self.square()`?

Reply:
Good idea 👍